.husky/pre-commit

```diff
@@ -1 +1,4 @@
-yarn format-staged && yarn lint-staged
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Husky adds this when you let it make a file, so I figure it helps somehow (probably with cross-platform)